report_thesis/src/sections/results/visual_analysis.tex

```diff
@@ -1,51 +1,62 @@
-\subsubsection{Visual Analysis of \ce{SiO_2} Distribution in Folds}
-This section aims to provide a detailed visual and statistical analysis of the \ce{SiO_2} concentration distribution across different folds used in our custom k-fold data partitioning procedure.
-The purpose is to demonstrate the effectiveness of our strategy in maintaining balanced and representative datasets, which is crucial for developing robust predictive models.
-
-We focus specifically on \ce{SiO_2} as a representative example to illustrate our methodology.
-Similar analyses were conducted for other oxides; however, the plots and figures for these are omitted here for brevity.
+\subsection{Visual and Statistical Analysis of \ce{SiO_2} Distribution in Partitioned Data}\label{sec:visual_analysis}
+This section provides a detailed visualization and statistical analysis of the \ce{SiO_2} concentration distribution across data partitions, following the customized k-fold data partitioning procedure described in Section~\ref{subsec:validation_testing_procedures}.
+We perform this analysis to validate the consistency of our data partitioning method and to provide a visual understanding of the data distribution it creates.
+The analysis focuses on \ce{SiO_2} as a representative example.
+We have conducted similar analyses for other oxides, but they are omitted here for brevity.
+
+As discussed in Section~\ref{subsec:validation_testing_procedures}, it is crucial to determine an optimal value of $p$ for the data partitioning algorithm that minimizes the number of extreme values in the test set, while still maintaining its representativeness.
+Our general approach is to select the lowest value of $p$ that still ensures the test set contains no extreme values, while maximizing its general representativeness.
+This helps in avoiding an overestimation of the model's performance due to extreme values while keeping the test set reflective of the majority of the data distribution.
+We developed a web-based platform to evaluate the performance of the data partitioning algorithm for different values of $p$, as shown in Figure~\ref{fig:web_platform} in Appendix~\ref{subsec:web_platform}.
+Utilizing the platform, we conducted analyses for various values of $p$ and determined that an optimal value is $p=5\%$.
+Furthermore, the 5\% threshold has demonstrated efficacy across all oxides within our dataset, and consequently, it has been uniformly applied throughout this study.
+This methodology is adaptable and can be employed to ascertain an optimal value for $p$ tailored to various targets, depending upon the specific dataset in use.
+Finally, we use a $k=5$ for our data partitioning algorithm.
+This results in five folds, with four folds used for cross-validation training and one fold designated as the test set.
```

Comment:
Is there a reason to mention this again? This is the third time IIRC.

Reply:
This is an entirely different section, so it serves as a reminder for the user as context here. The sections are somewhat far apart, so I think it's OK to give a reminder.